Engine/ac/movelist.h

```diff
@@ -23,8 +23,17 @@ using namespace AGS; // FIXME later
 #define MAXNEEDSTAGES 256
 #define MAXNEEDSTAGES_LEGACY 40
 
+struct MoveListPoint
```

Comment:
Unless you need extra variables, there's already a Point struct in geometry.h; in theory it may be extended later if more parameters will be needed.

Reply:
Made the switch to Point